lib/testing_appserver.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'onlyoffice_webdriver_wrapper'
```

Comment:
I think all requires of lib should be at one place - here or at `apserver_test_instance`